lib/gtfs/actions/general.js

```diff
@@ -44,7 +44,15 @@ const receivedStopsAndRoutes = createAction(
 export type GtfsGeneralActions = ActionType<typeof receivedGtfsElements> |
   ActionType<typeof receivedStopsAndRoutes>
 
-export function fetchStopsAndRoutes (entities: Array<any>, moduleType: string) {
+/**
+ * Fetch the specified stop and route entities for the alerts and sign modules.
+ * Module type indicates which module this is for (and which reducer should
+ * handle the result).
+ */
+export function fetchStopsAndRoutes (
+  entities: Array<{entity: {AgencyId: string, RouteId?: string, StopId?: string}}>,
+  moduleType: string
```

Comment:
Can you make the module type a union of possible strings?

Reply:
Does the type system have no support for enums?

Reply:
A union of any amount of types is allowed: https://flow.org/en/docs/types/unions/

Reply:
Thanks for the link, and sorry for any strange questions, I'm just trying to make sure I understand what I'm seeing. I am not accustomed to thinking of individual strings as types, but rather values of the type String - I didn't get the idea of unioning elements rather than sets. But I now see that Flow regards every literal value as its own type (https://flow.org/en/docs/types/literals/). I guess the literal "Blah" is a sort of syntactic shortcut for a set containing that one value `{"Blah"}`